app/client/components/identity/PublicProfile/index.tsx

```diff
@@ -112,7 +110,7 @@ export const PublicProfile = (_: { path?: string }) => {
       <WithStandardTopMargin>
         {error ? <GenericErrorMessage ref={errorRef} /> : null}
       </WithStandardTopMargin>
-      {loading ? loader : content()}
+      {user ? content(user) : loader}
```

Comment:
changes to this file seem a little extreme to resolve a little more type strictness, adding @paddyo to the reviewers

Reply:
Thanks. I had to specify `user`'s type to be `User | undefined` and `loading`'s value is only true when `user` is `undefined`. This felt redundant so I refactored it a little bit. Leaving the campsite tidier type of thing. The remaining changes are just so the type-checker understands that `user` is not used unless it is not `undefined`.

Reply:
This might actually be helpful in the near future. As part of our planned migration from tslint to eslint, the default linting configuration is stricter. I guess we could remove some of that strictness in the config, but I think I prefer it (obviously open to discussion :) )

Reply:
I think these changes look good. There was a reason we had a separate loader/user thing but it may have become redundant as the pages took shape when being built because it doesn't look needed any more and the refactor makes sense.